addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -108,6 +108,7 @@ var ActivityMenu = Widget.extend({
      */
     _onActivityActionClick: function (ev) {
         ev.stopPropagation();
+        this.$('.dropdown-toggle').dropdown('toggle');
```

Comment:
Menu does not close because of `ev.stopPropagation()`.
Should remove that line of code instead.

Reply:
@alexkuhn As far as I know, stopPropagation is intentionally done here, so we need to explicitly toggle dropdown

Reply:
I see, it's to prevent calling `_onActivityFilterClick`. Looks good then.